src/Value/Expression.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Sabberworm\CSS\Value;
+
+use Sabberworm\CSS\OutputFormat;
+use Sabberworm\CSS\Parsing\ParserState;
+
+/**
+ * An `Expression` represents a special kind of value that is comprised of multiple components wrapped in parenthesis.
+ * Examle `height: (vh - 10);`.
+ */
+class Expression extends CSSFunction
+{
+    /**
+     * @param ParserState $oParserState
+     * @param bool $bIgnoreCase
+     *
+     * @return Expression
+     *
+     * @throws SourceException
+     * @throws UnexpectedEOFException
+     * @throws UnexpectedTokenException
+     */
+    public static function parse(ParserState $oParserState, $bIgnoreCase = false)
+    {
+        $oParserState->consume('(');
+        $aArguments = self::parseArgs($oParserState);
+        $mResult = new Expression("", $aArguments, ',', $oParserState->currentLine());
+        $oParserState->consume(')');
+        return $mResult;
+    }
+}
```

Comment:
The should be a class-specific unit test for this class.  The overall parser tests are good, but we also want to make sure each individual class behaves as it should.

Reply:
@raxbg, would you be willing to create a corresponding `TestExpression` class to test the new class as a standalone?

Reply:
> There should be a class-specific unit test for this new class. The overall parser tests are good, but we also want to make sure each individual class behaves as it should.

Makes sense. How does that look like: 14118a20cee6273a33087b305f026aefcea77989